parkkihubi/settings.py

```diff
@@ -214,3 +214,5 @@
 PARKKIHUBI_ENFORCEMENT_API_ENABLED = (
     env.bool('PARKKIHUBI_ENFORCEMENT_API_ENABLED', True))
 PARKKIHUBI_PERMITS_PRUNABLE_AFTER = timedelta(days=3)
+PARKKIHUBI_NONE_END_TIME_REPLACEMENT = env.str(
+    'PARKKIHUBI_NONE_END_TIME_REPLACEMENT', default='2030-12-31T23:59:59Z')
```

Comment:
Move this new setting to line 210 just before all the API_ENABLED settings. That's more logical place for it, since it's related to the times.